plugin/src/main/java/org/opensearch/ml/processor/MLInferenceSearchResponseProcessor.java

```diff
@@ -774,6 +802,21 @@ public MLInferenceSearchResponseProcessor create(
                         + ". Please adjust mappings."
                 );
             }
+            boolean writeToSearchExtension = false;
+
+            if (outputMaps != null) {
+                for (Map<String, String> outputMap : outputMaps) {
+                    for (String key : outputMap.keySet()) {
+                        if (key.startsWith(EXTENSION_PREFIX)) {
+                            writeToSearchExtension = true;
+                            break;
+                        }
+                    }
+                }
+            }
+            if (writeToSearchExtension & oneToOne) {
+                throw new IllegalArgumentException("Writing model response to search extension does not support when one_to_one is true.");
```

Comment:
What do you feel about this?
```
Writing model response to search extension is not supported when one_to_one is true
```
or
```
Write model response to search extension is not supported when one_to_one is true
```

Reply:
will take the second one, it seems more formal this way